docs/src/main/asciidoc/http-reference.adoc

```diff
@@ -556,3 +556,32 @@ link:https://undertow.io/undertow-docs/undertow-docs-2.0.0/index.html#predicates
 === web.xml
 
 If you are using a `web.xml` file as your configuration file, you can place it in the `src/main/resources/META-INF` directory.
+
+== References
+
+=== Well-known route order values
+
+Route order marks are the values that are specified via Vert.x route `io.vertx.ext.web.Route.order(int)` function.
```

Comment:
Is "order mark" some established term, or? If not I would stick with "order value" instead.

Reply:
No, ‘mark’ surprised me too.

Reply:
Me too, took it from the comments in `VertxHttpRecorder`, but `order value` sounds better, will rephrase.